builddefs/message.mk

```diff
@@ -87,6 +87,7 @@ define GENERATE_MSG_AVAILABLE_KEYMAPS
 endef
 MSG_AVAILABLE_KEYMAPS = $(eval $(call GENERATE_MSG_AVAILABLE_KEYMAPS))$(MSG_AVAILABLE_KEYMAPS_ACTUAL)
 
+MSG_BOOTLOADER_NOT_FOUND_BASE = Bootloader not found. Make sure the board is in DFU mode. See https://docs.qmk.fm/\#/newbs_flashing\n
```

Comment:
Some bootloaders are not "DFU" bootloaders, this might be more confusing tbh.

Reply:
Updated wording. As you may have guesses from the nature of this PR, I'm a bit new to this (and the intent of this PR is to guide other newbies a bit better).